Specs/DateTime/Portuguese/DateTimeModel.json

```diff
@@ -3120,5 +3120,535 @@
+              "value": "2021-12-03"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "vamos nos encontrar na terça qua qui ou sexta",
+    "Context": {
+      "ReferenceDateTime": "2021-11-29T12:00:00"
+    },
+    "NotSupported": "java, javascript, python",
+    "Results": [
+      {
+        "Text": "terça",
+        "Start": 23,
+        "End": 27,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-WXX-2",
+              "type": "date",
+              "value": "2021-11-23"
+            },
+            {
+              "timex": "XXXX-WXX-2",
+              "type": "date",
+              "value": "2021-11-30"
+            }
+          ]
+        }
+      },
+      {
+        "Text": "qua",
+        "Start": 29,
+        "End": 31,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-WXX-3",
+              "type": "date",
+              "value": "2021-11-24"
+            },
+            {
+              "timex": "XXXX-WXX-3",
+              "type": "date",
+              "value": "2021-12-01"
+            }
+          ]
+        }
+      },
+      {
+        "Text": "qui",
+        "Start": 33,
+        "End": 35,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-WXX-4",
+              "type": "date",
+              "value": "2021-11-25"
+            },
+            {
+              "timex": "XXXX-WXX-4",
+              "type": "date",
+              "value": "2021-12-02"
+            }
+          ]
+        }
+      },
+      {
+        "Text": "sexta",
+        "Start": 40,
+        "End": 44,
+        "TypeName": "datetimeV2.date",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "XXXX-WXX-5",
+              "type": "date",
+              "value": "2021-11-26"
+            },
+            {
+              "timex": "XXXX-WXX-5",
+              "type": "date",
+              "value": "2021-12-03"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "vou voltar as 15 para as 8 de amanhã",
+    "Context": {
+      "ReferenceDateTime": "2020-05-14T12:00:00"
+    },
+    "NotSupported": "javascript, java",
+    "Results": [
+      {
+        "Text": "15 para as 8 de amanhã",
+        "Start": 14,
+        "End": 35,
+        "TypeName": "datetimeV2.datetime",
+        "Resolution": {
+          "values": [
+            {
+              "timex": "2020-05-15T07:45",
+              "type": "datetime",
+              "value": "2020-05-15 07:45:00"
+            },
+            {
+              "timex": "2020-05-15T19:45",
+              "type": "datetime",
+              "value": "2020-05-15 19:45:00"
+            }
+          ]
+        }
+      }
+    ]
+  },
+  {
+    "Input": "vou voltar amanhã 8 horas am",
```

Comment:
Can you add  cases for "vamos estar de volta amanhã 8 horas da manhã" and "vamos estar de volta amanhã de 8 horas da manhã"?

Reply:
added